README.md

```diff
@@ -35,19 +35,40 @@ See [Pyspark](examples/Pyspark Kernel.ipynb) and [Spark](examples/Spark Kernel.i
 
 1. Install the library
 
-        pip install sparkmagic
+        `pip install sparkmagic`
```

Comment:
With the way you have it formatted, the backticks show up in the document which I don't think is what you wanted.

![image](https://cloud.githubusercontent.com/assets/13972471/18284024/100235a8-741d-11e6-868d-7dde621dc2ed.png)